phalcon/debug/dump.zep

```diff
@@ -193,10 +195,21 @@ class Dump
 				}
 			} else {
 				// Debug all properties
-				for key, value in variable {
+                do {
+
+                    let attr = each(variable);
```

Comment:
Could you please avoid `each` function. See: https://github.com/phalcon/cphalcon/pull/13308

Reply:
Sorry, I not find a method to replace each.So I remove detailed mode for `Phalcon\\Debug\\Dump `. When php version < 7.2, it works.

Reply:
Anyway this code will introduce `E_DEPRECATED` error

Reply:
I see, maybe someone can fix it later. I'll try it, too.

Reply:
Thanks